### DISCUSSION TITLE
Files with "changes since last view" no longer work

### DISCUSSION THREAD

**Discussion by RussKie** [community/community#16829] [Pull Requests]
It appears to started happening in the past few days - whenever I get to a PR that I previously reviewed and that has new changes, the files that have new changes are no longer expanded by default:
![image](https://user-images.githubusercontent.com/4403806/168943654-a94d5e8e-813e-40d0-8af4-971cbd840034.png)
I have disabled all extensions to confirm it is the GitHub's doing. 

Furthermore, once unmarked - it's impossible to mark a file back as "viewed" nor the marker goes away, nor the unmarked state is persisted!

https://user-images.githubusercontent.com/4403806/168945787-462a719a-ab30-4f19-89c8-47bf00e91cab.mp4

**Comment by timmy-circle** [comment]
having exactly the same issue.

**Comment by smoogipoo** [comment]
Also some files are minimised by default when creating or reviewing a PR.

**Comment by ysoares0209** [comment]
This was also reported here
https://github.com/github/feedback/discussions/16811
